assets/js/modules/analytics/util/report-normalization.js

```diff
@@ -51,3 +53,26 @@ const normalizeMetrics = ( metrics ) => {
 		.filter( ( metric ) => isPlainObject( metric ) )
 	;
 };
+
+const normalizeDimensions = ( dimensions ) => {
+	return castArray( dimensions )
+		.map(
+			( dimension ) => typeof dimension === 'string'
+				? { name: dimension }
+				: dimension
+		)
+		.filter( ( dimension ) => isPlainObject( dimension ) )
+	;
+};
```

Comment:
The normalization here makes sense, but this causes a bug visible on the dashboard, in `DashboardPopularPagesWidget`, where (for whatever odd reason 😆 ) we're passing multiple values as a comma-separated string. I'll update that component to pass the values as array of strings which is more appropriate.

Reply:
We initially had the normalisation to pass dimensions to the `isValidDimensionFilters`, which we don't need now, but it's a good idea to keep it as it makes it cleaner to have it normalised.